cli/lsp/testing/execution.rs

```diff
@@ -754,19 +755,14 @@ impl test::TestReporter for LspTestReporter {
     self.progress(lsp_custom::TestRunProgressMessage::Started { test });
   }
 
-  fn report_output(&mut self, output: &test::TestOutput) {
+  fn report_output(&mut self, output: &[u8]) {
     let test = self.current_origin.as_ref().and_then(|origin| {
       self
         .stack
         .get(origin)
         .and_then(|v| v.last().map(|td| td.into()))
     });
-    let value = match output {
-      test::TestOutput::String(value) => value.replace('\n', "\r\n"),
-      test::TestOutput::Bytes(bytes) => {
-        String::from_utf8_lossy(bytes).replace('\n', "\r\n")
-      }
-    };
+    let value = String::from_utf8_lossy(output).replace('\n', "\r\n");
```

Comment:
This is actually somewhat of a bug because the data is sent over buffered and so this might be between a multi-byte character... I'm not sure how to fix this.